.github/workflows/build-listing.yml

```
@@ -66,10 +66,10 @@ jobs:
           args: build Documentation/docfx.json
       - name: move the documentation to the publish directory
         run: cp -r Documentation/_site ${{ env.listPublishDirectory }}/docs
-      - name: Build Package Version Listing with Nuke
-        run: ${{ env.pathToCi }}/build.cmd BuildRepoListing --root ${{ env.pathToCi }} --list-publish-directory $GITHUB_WORKSPACE/${{ env.listPublishDirectory }} --current-package-name ${{ vars.PACKAGE_NAME }}
-        env:
+      - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        name: Build Package Version Listing with Nuke
+        run: ${{ env.pathToCi }}/build.cmd BuildRepoListing --root ${{ env.pathToCi }} --list-publish-directory $GITHUB_WORKSPACE/${{ env.listPublishDirectory }} --current-package-name ${{ vars.PACKAGE_NAME }}
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:100: Double quote to prevent globbing and word splitting

(shellcheck)